apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2/RowFilters.cs

```diff
@@ -177,6 +194,28 @@ public static class RowFilters
                 }
             };
 
+        /// <summary>
+        /// Creates a new <see cref="RowFilter"/> instance which matches only cells
+        /// from column families whose names match the exact string specified.
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// Note that string is implicitly convertible to <see cref="BigtableByteString"/>, so <paramref name="value"/> can
+        /// be specified using a string as well and its UTF-8 representations will be used.
+        /// </para>
+        /// </remarks>
+        /// <param name="value">
+        /// The exact value used to match column families. Must not be null
+        /// </param>
+        /// <returns>The created row filter.</returns>
+        public static RowFilter FamilyNameExact(string value) =>
+            new RowFilter
```

Comment:
I'd really like to avoid the conversion to UTF-8 and back again if possible. Let's talk about that when the big "iterate over bytes" issue is sorted - it may drop out of that.

Reply:
I assumed since this is unlikely to be a source of performance bottlenecks, it would be better to have a single escape helper operate over byte strings rather than a duplication of logic. But I can change it if you feel otherwise.

Reply:
Happy to leave it for now. It's an implementation detail after all - we can fix it later if we see issues.